libs/blocks/icon-block/icon-block.css

```diff
@@ -177,6 +177,11 @@
   margin-top: var(--spacing-s);
 }
 
+.icon-block .foreground .second-column .title-row {
+  display: flex;
+  align-items: center;
+}
```

Comment:
Since you're already using flex, couldn't a `column-gap` be used instead of the more complex margin logic?

Reply:
This code is primarily used to fix the text under the icon.
Yes, `column-gap` will solve this problem on pages written by **ICON BLOCK**. However, if RTL is enabled, it will not fix icon spacing on other pages.
The goal is to solve the **no space** problem on all milo pages. As a result, we changed the margin in **icon.js** to `margin-inline`, allowing the margin to be added based on direction.